services/common/src/redux/utils/Validate.ts

```diff
@@ -277,11 +277,15 @@ export const dateInFuture = (value) =>
   value && !moment(value).isAfter() ? "Date must be in the future" : undefined;
 
 // NOTE: modified from version in CORE- change from <= to <
-export const dateNotBeforeOther = memoize((other: string) => (value: string) => {
-  return value && other && value < other
-    ? `Date cannot be before ${moment(other).format("ddd MMM D YYYY")}`
-    : undefined;
-});
+export const dateNotBeforeOther = memoize((other: string, otherLabel?: string) => (value: string) => {
+  const invalid = value && other && value < other;
+  if (!invalid) {
+    return undefined;
+  }
+  const otherFormattedDate = moment(other).format("ddd MMM D YYYY");
```

Comment:
I'd love to see these converted to using DateFns instead of moment at some point.  But I think we have a ticket to throw moment in the garbage already, so maybe it can wait for that.